src/components/Navbar.tsx

```diff
@@ -15,12 +15,14 @@ export function DevNavbar({ children }: { children: React.ReactNode }) {
       backgroundColor="brand.50"
       alignItems="center"
       px={6}
+      boxShadow="0px 1px 0px #29262F"
```

Comment:
Can we add this to the color palette under `theme` so it's easier to modify later?

Reply:
I added it while resolving the merge conflicts.